db/migrate/20160907092724_create_teams_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateTeamsUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :teams_users do |t|
+      t.belongs_to :team, index: true
+      t.belongs_to :user, index: true
+    end
```

Comment:
we need an composite index that validates the uniqueness of the user-id , team_id (so in db can't be 2 team_users with the same user and team)